libs/datamodel/core/src/validator/validate.rs

```diff
@@ -287,7 +287,7 @@ impl<'a> Validator<'a> {
         if super::invalid_model_names::RESERVED_MODEL_NAMES.contains(&model.name.as_str()) {
             Err(DatamodelError::new_model_validation_error(
                 &format!(
-                    "The model name `{}` is invalid. It is a reserved name. Please change it.",
+                    "The model name `{}` is invalid. It is a reserved name. Please change it. Read more at https://pris.ly/d/naming-models",
```

Comment:
Checked out the link. It does not say anything about reserved names. A user might not realize they are supposed to use `@@map` now. So maybe the message could include that. E.g.
> Please change it and use `@@map` if you want to keep this in the underlying database. Read more at https://pris.ly/d/naming-models